application.py

```diff
@@ -703,6 +703,27 @@ def resource_by_template(primary_key_value=None):
 
         return full_rsp
 
+@application.route('/articles', methods=['GET'])
+def get_articles():
+    try:
+        #TODO change to request[password] and do jwt.decode()
+        password = "Yunho123"
+        user = UsersRDB.find_user(password)
+        results = UsersRDB.find_postinfo(user)
+
+        rsp_status = 504
```

Comment:
if this is for a successfully processed request, the response should have status of 200